app/tests/spec/lib/router.js

```diff
@@ -399,6 +399,13 @@ function (chai, sinon, Backbone, Router, SignInView, SignUpView, ReadyView,
         assert.isFalse(router.navigate.called);
       }
 
+      function setUpIFrameLink() {
+        sinon.stub(router.environment, 'isFramed', function () {
```

Comment:
Instead of accessing `router.environment` directly, I would create and pass in an `Environment` object to the Router constructor, then do:

```js
sinon.stub(environment, 'isFramed', function () {
```

This is a subtle change, but the change makes fewer assumptions about what should be internal implementation details of router and the name where the environment object is stored.